app/gateway/2.8.x/reference/external-plugins.md

```diff
@@ -272,7 +273,7 @@ pluginserver_other_one_query_cmd = /usr/local/bin/other-one -dump
 
 {{site.base_gateway}} support for the JavaScript language is provided by [kong-js-pdk].
 The library provides a plugin server to provide runtime for JavaScript plugins, and
-functions to access {{site.base_gateway}} features of the [PDK][kong-pdk].
+functions to access {{site.base_gateway}} features of the [PDK](/gateway/latest/pdk/).
```

Comment:
This uses the markdown anchor reference syntax. We can update the `[kong-pdk]` reference at the bottom of the file and leave these as-is, or update these and remove the reference at the bottom.

For consistency with other URLs on the page, I'd recommend leaving the reference in place and updating line 633 only